app/api/ada/index.test.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import './lib/test-config';
+import AdaApi from './index';
+import { RustModule } from './lib/cardanoCrypto/rustLoader';
+import {
+  silenceLogsForTesting,
+} from '../../utils/logging';
+
+import type {
+  FilterUsedRequest,
+  FilterUsedResponse,
+} from './lib/state-fetch/types';
+
+beforeAll(async () => {
+  await RustModule.load();
+  //silenceLogsForTesting();
+});
+
+test('Restore wallet', async () => {
+  async function checkAddressesInUse(_body: FilterUsedRequest): Promise<FilterUsedResponse> {
+    return [];
+  }
+
+  const restoreRequest = {
+    recoveryPhrase: 'prevent company field green slot measure chief hero apple task eagle sunset endorse dress seed',
+    walletName: 'mywallet',
+    walletPassword: '123',
+    checkAddressesInUse
+  };
+
+  const response : any = await AdaApi.prototype.restoreWallet(restoreRequest);
```

Comment:
Remember we need this function call to not modify anything in local storage.
The restoreWallet function you call contains both
```
saveCryptoAccount(cryptoAccount);
createStoredWallet(adaWallet, masterKey);
```
so you will need to either modify `restoreWallet` OR create a new function that doesn't save to storage

Reply:
nevermind, realized this is meant to just add a test for the existing functionality